src/client/Microsoft.Identity.Client.Broker/WamAdapters.cs

```diff
@@ -150,11 +151,11 @@ private enum ResponseStatus
         /// Gets the Common Auth Parameters to be passed to Native Interop
         /// </summary>
         /// <param name="authenticationRequestParameters"></param>
-        /// <param name="isMsaPassthrough"></param>
+        /// <param name="brokerOptions"></param>
         /// <param name="logger"></param>
         public static NativeInterop.AuthParameters GetCommonAuthParameters(
             AuthenticationRequestParameters authenticationRequestParameters, 
-            bool isMsaPassthrough,
+            WindowsBrokerOptions brokerOptions,
```

Comment:
is this a breaking change?

Reply:
It shouldn't be, earlier we were only using WindowsBrokerOptions for MSA-PT in `GetCommonAuthParameters`, now that we are also passing HeaderTitle to Interop, modified the method to accept `WindowsBrokerOptions` instead.